spec/lang/syntax.md

```diff
@@ -419,10 +427,12 @@ pub enum IntrinsicOp {
 Finally, the general structure of programs and functions:
 
 ```rust
-/// Opaque types of names for functions and globals.
+/// Opaque types of names for functions, vtables, trait methods and globals.
```

Comment:
```suggestion
/// Opaque types of names for functions, vtables, trait methods, and globals.
```

Reply:
So, you choose to go for the path of Oxford Commas 😂